src/subtyping.md

```diff
@@ -152,38 +133,36 @@ A type constructor F's *variance* is how the subtyping of its inputs affects the
-* `F` is *invariant* otherwise (no subtyping relationship exists)
+* F is **covariant** if `F<Sub>` is a subtype of `F<Super>` (the subtype property is passed through)
+* F is **contravariant** if `F<Super>` is a subtype of `F<Sub>` (the subtype property is "inverted")
+* F is **invariant** otherwise (no subtyping relationship exists)
 
-If `F` has multiple type parameters, we can talk about the individual variances
-by saying that, for example, `F<T, U>` is covariant over `T` and invariant over `U`.
+If we remember from the above examples,
+it was ok for us to treat `&'a T` as a subtype of `&'b T` if `'a: 'b`,
+therefore we can say that `&'a T` is *covariant* over `'a`.
 
-It is very useful to keep in mind that covariance is, in practical terms, "the"
-variance. Almost all consideration of variance is in terms of whether something
-should be covariant or invariant. Actually witnessing contravariance is quite difficult
-in Rust, though it does in fact exist.
+Also, we saw that it was not ok for us to treat `&mut &'a T` as a subtype of `&mut &'b T`,
+therefore we can say that `&mut T` is *invariant* over `T`
 
-Here is a table of important variances which the rest of this section will be devoted
-to trying to explain:
+Here is a table of some other type constructors and their variances:
 
 |   |                 |     'a    |         T         |     U     |
 |---|-----------------|:---------:|:-----------------:|:---------:|
-| * | `&'a T `        | covariant | covariant         |           |
-| * | `&'a mut T`     | covariant | invariant         |           |
-| * | `Box<T>`        |           | covariant         |           |
+|   | `&'a T `        | covariant | covariant         |           |
+|   | `&'a mut T`     | covariant | invariant         |           |
+|   | `Box<T>`        |           | covariant         |           |
 |   | `Vec<T>`        |           | covariant         |           |
-| * | `UnsafeCell<T>` |           | invariant         |           |
+|   | `UnsafeCell<T>` |           | invariant         |           |
 |   | `Cell<T>`       |           | invariant         |           |
-| * | `fn(T) -> U`    |           | **contra**variant | covariant |
+|   | `fn(T) -> U`    |           | **contra**variant | covariant |
 |   | `*const T`      |           | covariant         |           |
 |   | `*mut T`        |           | invariant         |           |
 
-The types with \*'s are the ones we will be focusing on, as they are in
-some sense "fundamental". All the others can be understood by analogy to the others:
+Some of these can be explained simply in relation to the others:
 
 * `Vec<T>` and all other owning pointers and collections follow the same logic as `Box<T>`
 * `Cell<T>` and all other interior mutability types follow the same logic as `UnsafeCell<T>`
+* `UnsafeCell<T>` having interior mutability gives it the same variance properties as `&mut T`
```

Comment:
Can you think of a good way to explain that?

The original mostly just says that since `UnsafeCell<T>` allows you to treat a `&T` as a `&mut T`, it should have the same invariance